ui/src/tests/People.test.tsx

```diff
@@ -41,21 +41,21 @@ describe('people actions', () => {
         TestUtils.mockClientCalls();
     });
 
-    it('opens PersonForm component in editing mode when hambaga icon is clicked', async () => {
-        const app = renderWithRedux(<PeopleMover/>);
+    // it('opens PersonForm component in editing mode when hambaga icon is clicked', async () => {
```

Comment:
Instead of commenting these out, can we just put an x before the it statement so that it at least tells you that there are tests that were ignored?